docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -0,0 +1,49 @@
+# XMR<>BTC Atomic Swap - User Interface Prototype Validation
+
+This document sums up the questions that we would like to validated with 
+
+## Questions
+
+With the UI prototypes we are trying to answer the following questions.
+The questions are split between maker and taker, because the objectives are somewhat different.
+
+Is a `taker` happy to...
+
+1. run a bitcoind full-node
+1. run a monerod full-node
+1. use a webpage for discovering makers
+1. use a privacy chat tool for discovering makers
+1. manually add makers to retrieve orders
+1. download software (swap execution daemon) before being able to do a swap
+1. run the swap daemon locally on the taker's machine
+1. keep the swap daemon running for the whole length of the swap execution
+1. give control over the bitcoin and monero wallet to the swap execution daemon
+1. keep the browser open for the whole length of the swap execution
+1. be in different cryptographic roles depending on the direction of the swap
+1. run the swap execution daemon without using Tor
+1. connect to a maker that does not use Tor
```

Comment:
```suggestion
1. to not use tor
```

I would not separate both.

Makes me think, in the mocks, we need to be sure to add an address for the maker on both CLI and swap UI (and this address can be ip4 to validate this assumption).

Reply:
Yes! I'll adapt what is currently there to show ip4 addresses. 

Note:
In the long run we might want use more, simple mocks that focus on specific questions rather than one big flow. I don't think we can expect users to focus on too many small details. Alternatively I could also add more stickies to point out question points. 
Not sure what is the best way there yet.